src/main/java/com/ode22/catnews_origins/Client/CatClient.java

```diff
@@ -0,0 +1,44 @@
+package com.ode22.catnews_origins.Client;
+
+import java.io.*;
+import java.net.*;
+import java.util.List;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.ode22.catnews_origins.Dto.RandomCat;
+
+public class CatClient
```

Comment:
Check https://github.com/JohannaGaudeck/CatNews_origins/blob/e070f8a986ecea12a25fb5f1d55e978f676e3332/src/main/java/com/ode22/catnews_origins/Client/ApaClient.java found a few ways to improve the code